packages/net/ip/address.pony

```diff
@@ -1,3 +1,5 @@
+use "net/dns"
+
 class val IPAddress
```

Comment:
I think this file should be named `ip_address.pony`, because the principal type declared in the file is `IPAddress`.